privacy_evaluator/metrics/privacy_risk_score.py

```diff
@@ -0,0 +1,72 @@
+from privacy_evaluator.classifiers.classifier import Classifier
+import numpy as np
+
+
+def compute_privacy_risk_score(
+    target: Classifier,
+    x_train: np.ndarray,
+    y_train: np.ndarray,
+    x_test: np.ndarray,
+    y_test: np.ndarray,
+    num_bins=15,
+):
+    """
+    Computes each individual point's likelihood of being a member
+    (denoted as privacy risk score in https://arxiv.org/abs/2003.10595).
+
+    For an individual sample, its privacy risk score is computed as the posterior
+    probability of being in the training set
+    after observing its prediction output by the target machine learning model.
+
+    (Helper method and description taken from
+    https://github.com/tensorflow/privacy/blob/master/tensorflow_privacy/privacy/membership_inference_attack/membership_inference_attack.py#L217)
+
+
+    :param attack_input: input data for compute membership probability
+    :param num_bins: the number of bins used to compute the training/test histogram
+    :return: membership probability results
+    """
+    loss_train = target.art_classifier.compute_loss(x_train, y_train)
+    loss_test = target.art_classifier.compute_loss(x_test, y_test)
+    return _compute_membership_probability(loss_train, loss_test, num_bins)
+
+
+def _compute_membership_probability(loss_train, loss_test, num_bins: int = 15):
```

Comment:
Can you please add the return types here (e.g. `_compute_membership_probability(loss_train, loss_test, num_bins: int = 15) -> Tupel[np.ndarray, np.ndarray]:`)